src/index.js

```diff
@@ -10,9 +10,13 @@ const REGEX_LOWERCASE = /[a-z]+/;
 const REGEX_UPPERCASE = /[A-Z]+/;
 const REGEX_NUMBERS = /\d+/;
 const REGEX_SPECIAL = /[^A-Za-z\d]+/;
+const REGEX_TIME = /^\d{1,2}:\d{1,2}$/;
```

Comment:
why not \d{2} always?

Reply:
can be 1:5

Reply:
that is invalid time, we allow to input that but it shouldn't validate